rabbitmq/tests/test_unit.py

```diff
@@ -13,7 +13,10 @@
 
 from . import common, metrics
 
-pytestmark = pytest.mark.unit
+pytestmark = [
+    pytest.mark.unit,
+    pytest.mark.skipif(common.METRICS_PLUGIN == "prometheus", reason="Not testing management plugin metrics."),
+]
 
 
 @pytest.mark.unit
```

Comment:
nit - No need to repeat `@pytest.mark.unit` if you use `pytestmark`, you can remove them